### PR TITLE
Add dropdown to installation section for installing from source.

### PR DESCRIPTION
**Pull request by unaidedelf8777**
added a short section for installing from source.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,15 +42,45 @@ Lilac runs **on-device** using open-source LLMs with a UI and Python API for:
 ## 🔥 Getting started
 
 ### 💻 Install
-
+*Install via pip*
 ```sh
 pip install lilac[all]
 ```
-
+*On HiggingFace Spaces*
```
Comment (dsmilkov) [inline README.md:49]: small typo: HiggingFace -> HuggingFace

File: README.md [hunk 2]
```diff
@@ -42,15 +42,45 @@ Lilac runs **on-device** using open-source LLMs with a UI and Python API for:
...
 If you prefer no local installation, you can fork the
```
Comment (dsmilkov) [inline README.md:50]: can you add a newline between the section title (line 49) and this (line 50) so markdown renders the title in a seperate line

File: README.md [hunk 3]
```diff
@@ -42,15 +42,45 @@ Lilac runs **on-device** using open-source LLMs with a UI and Python API for:
...
 [fork the HuggingFace Spaces demo](https://lilacai-lilac.hf.space/). Documentation
 [here](https://docs.lilacml.com/huggingface/huggingface_spaces.html).
 
+<details>
+  <summary>*Install from source*</summary>
+  
+  first, clone the repo:
```
Comment (dsmilkov) [inline README.md:57]: small typo: capitalize "First"

File: README.md [hunk 4]
```diff
@@ -42,15 +42,45 @@ Lilac runs **on-device** using open-source LLMs with a UI and Python API for:
...
+  ```sh
+  git clone https://github.com/lilacai/lilac.git
+  ```
+  Now, Build from source:
```
Comment (dsmilkov) [inline README.md:61]: lowercase "build"

File: README.md [hunk 5]
```diff
@@ -42,15 +42,45 @@ Lilac runs **on-device** using open-source LLMs with a UI and Python API for:
...
+  > Note: you must have atleast the lts version of `nodejs` and `npm` installed to build.
```
Comment (dsmilkov) [inline README.md:62]: space between "at" and "least"

**Review by dsmilkov — COMMENTED** [review]
This is awesome, just a couple of tiny typos

**Comment by nsthorat** [comment]
@dsmilkov lets merge this and then make the edits ourselves since they're tiny?

**Comment by dctanner** [comment]
`poetry install` should be run before `poetry shell`

**Comment by dctanner** [comment]
I also had to do `poetry add jinja2` for ./scripts/build_server_prod.sh to run correctly, as the dep is missing.

**Pull request closed without merging**